sds_in_a_box/SDSCode/requirements.txt

```diff
@@ -1 +1,3 @@
+requests
```

Comment:
What is this requirements.txt used for?

Reply:
When you use a that new CDK function to build a python lambda function, it automatically installs the libraries listed in the requirements.txt

Reply:
This one here: https://docs.aws.amazon.com/cdk/api/v2/python/aws_cdk.aws_lambda_python_alpha/README.html